lib/rules/no-expose-after-await.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
     fixable: null,
     schema: [],
     messages: {
-      forbidden: 'The `expose` after `await` expression are forbidden.'
+      forbidden: 'The `{{name}}` after `await` expression are forbidden.'
```

Comment:
Better wording (might also apply to similar `no-*-after-await` rules):

```suggestion
      forbidden: '`{{name}}` is forbidden after an `await` expression.'
```